tensorflow/compiler/xla/service/cpu/compiler_functor.cc

```diff
@@ -211,6 +214,11 @@ void CompilerFunctor::AddOptimizationPasses(
   builder.SLPVectorize = opt_level_ > 1;
 
   builder.populateFunctionPassManager(*function_passes);
+
+  llvm::PassRegistry &Registry = *llvm::PassRegistry::getPassRegistry();
+  ::polly::initializePollyPasses(Registry);
+  ::polly::registerPollyPasses(*module_passes);
```

Comment:
Sorry, my bad.  I meant to say `llvm::PassRegistry& registry = *llvm::PassRegistry::getPassRegistry();`.